aiopyrq/queues.py

```diff
@@ -108,23 +108,25 @@ async def add_items(self, items: list):
         await pipeline.execute()
         await self._wait_for_synced_slaves()
 
-    async def delete_item(self, item) -> None:
+    async def delete_item(self, item, count: int = 0) -> None:
```

Comment:
tady a i nize me jen napada, ze kdyz das nazev count, tak je to pocet a v pripade 0 to vypada, jakoby nesmaz nic. Ale pritom to funguje, smaz vse, zda to alespon nenapsat do popisku te promenne. In case count is zero all items to be deleted ?